composer.json

```diff
@@ -4,53 +4,61 @@
     "type": "project",
     "license": "AGPL-3.0-only",
     "require": {
-        "php": "^7.4",
+        "php": "^8.1",
         "ext-ctype": "*",
         "ext-iconv": "*",
         "ext-json": "*",
         "api-platform/core": "^2.6",
         "cloudinary/cloudinary_php": "^2.7",
-        "danskernesdigitalebibliotek/agency-auth-bundle": "^2.0",
+        "danskernesdigitalebibliotek/agency-auth-bundle": "^3.0",
         "doctrine/annotations": "^1.10",
         "doctrine/doctrine-bundle": "^2.5",
         "doctrine/doctrine-migrations-bundle": "^3.0",
         "doctrine/orm": "^2.11",
         "nelmio/cors-bundle": "^2.2",
         "phpdocumentor/reflection-docblock": "^5.3",
         "phpstan/phpdoc-parser": "^1.2",
```

Comment:
phpstan  package? Should it have been moved to required-dev?

Also phpdocumentor/reflection-docblock can be removed.